include/raygizmo.h

```diff
@@ -124,12 +124,18 @@ void main() { \
 #define PICKING_FBO_WIDTH 512
 #define PICKING_FBO_HEIGHT 512
 
+#if defined(__cplusplus) || _MSC_VER
```

Comment:
Can we utilize raylib's `CLITERAL` macro for this? And for other similar stuff below